Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:carbon
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
```

Comment:
You can be specific with both the files.

Reply:
You mean like:
COPY package*.json ./package*.json
right?
Ok

Reply:
Yes

Reply:
Ack
Done